quaint/src/connector/postgres/native/mod.rs

```diff
@@ -285,6 +289,12 @@ impl PostgreSql {
             }
         }
 
+        let system_name = if is_cockroachdb {
+            SYSTEM_NAME_COCKROACHDB
+        } else {
+            SYSTEM_NAME_POSTGRESQL
+        };
```

Comment:
In case of postgres, this can be a method instead:

```rust
fn db_system_name(&self) -> &'static str {
    if self.is_cockroachdb {
        DB_SYSTEM_NAME_COCKROACHDB
    } else {
        DB_SYSTEM_NAME_POSTGRESQL
    }
}
```

But I guess storing one fat pointer in the struct is also acceptable if you want to avoid branching.